system-int-test/test/cmr/system_int_test/search/collection_spatial_search_test.clj

```diff
@@ -464,7 +473,31 @@
         :point [17.73 2.21] [whole-world normal-brs]
         :line [-0.37,-14.07,4.75,1.27,25.13,-15.51]
         [whole-world polygon-with-holes polygon-with-holes-cart normal-line-cart normal-line
-         normal-poly-cart]))))
+         normal-poly-cart]))
+
+    (testing "ORed spatial search"
+       (let [poly-coordinates ["-16.79,-12.71,-6.32,-10.95,-5.74,-6.11,-15.18,-7.63,-16.79,-12.71"
+                               "0.53,39.23,21.57,59.8,-112.21,84.48,-13.37,40.91,0.53,39.23"]
+             poly-refs (search/find-refs
+                        :collection
+                        {:polygon poly-coordinates
+                         "options[spatial][or]" "true"})
+             poly-items [wide-north on-np normal-poly very-wide-cart whole-world normal-brs]
+             bbox-refs (search/find-refs
+                               :collection
+                               {:bounding-box ["166.11,-19.14,-166.52,53.04"
+                                               "23.59,-15.47,25.56,-4"]
+                                              "options[spatial][or]" "true"})
+             bbox-items [across-am-poly very-wide-cart am-point along-am-line normal-line-cart whole-world across-am-br]
+             combined-refs (search/find-refs
+                            :collection
+                            {:circle "179.8,41,100000"
+                             :bounding-box "166.11,-19.14,-166.52,53.04"
+                             "options[spatial][or]" "true"})
+             combined-items [across-am-poly along-am-line whole-world across-am-br am-point very-wide-cart]]
+        (is (d/refs-match? combined-items combined-refs))
```

Comment:
nit: it's better to have the expected items explicitly listed in the verification statements rather than have it far away in the `let` block.